spec/lib/audit/catalog_to_moab_class_spec.rb

```diff
@@ -89,4 +89,46 @@
       subject
     end
   end
+
+  context '.check_version_druid' do
+    include_context 'fixture moabs in db'
+    let(:druid) { 'bj102hs9687' }
+    let(:subject) { described_class.check_version_druid(last_checked_version_b4_date, druid, limit) }
+
+    context 'when there are PreservedCopies to check' do
+      let(:c2m_mock) { instance_double(described_class) }
+
+      it 'creates an instance and calls #check_catalog_version for every result when results are in a single batch' do
```

Comment:
same copy pasta brain freeze.   Seems like you may want to affirm that you process each preserved copy associated with the druid.  batches ... not so much.